apps/yarn.lock

```diff
@@ -60,7 +60,7 @@
     temporal latest
   optionalDependencies:
     browser-serialport latest
-    firmata "^0.12.0"
+    firmata latest
```

Comment:
Looks like a workaround we wrote [to not register the CP_COMMAND response more than once](https://github.com/code-dot-org/playground-io/pull/1) is not working with the latest version of Firmata - tracking down exactly why and fixing...